install.rb

```diff
@@ -294,6 +308,12 @@ def prepare_installation
     bindir = RbConfig::CONFIG['bindir']
   end
 
+  if not InstallOptions.localedir.nil?
+    localedir = InstallOptions.localedir
+  else
+    localedir = "/opt/puppetlabs/puppet/share/puppet/locale"
```

Comment:
It looks like some of the other directories/files installed by `install.rb` include a Windows path. Does that apply here and if so are we saving the windows use-case for a later PR?